libqtile/layout/max.py

```diff
@@ -44,7 +45,7 @@ def add(self, client):
         return super().add(client, 1)
 
     def configure(self, client, screen_rect):
-        if self.clients and client is self.clients.current_client:
+        if not self.only_focused or (self.clients and client is self.clients.current_client):
```

Comment:
Can you add some reasoning for this change? Or better yet, can we drop it all together?

Reply:
That's basically the change that allows the `Max` layout to display multiple windows on top of each other. That's really nothing I would want to remove!

Reply:
Ok, but why is that necessary? Max should only ever display one window, maximized. It's very unintuitive.

Reply:
See #415 . The added code gives the user the option to display more than one window - only really useful when using transparency, but then it is very neat.

Reply:
Let's default it to off, then.

Reply:
Ok.